packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -10,67 +10,224 @@
 /* eslint-disable arrow-body-style */
 /* eslint-disable max-len */
 
-// Declare some globals that we'll need throughout
-let activeMeeting, remoteShareStream, webex;
 
-// First, let's wire our form fields up to localStorage so we don't have to
-// retype things everytime we reload the page.
+let redirect_uri;
+let activeMeeting;
+let remoteShareStream;
+let isToken;
+let isOAuth;
+let webex;
+
+function getOAuthRedirectURI() {
```

Comment:
Let's remove this function and just have this as part of the redirect_uri definition.

```js
let redirect_uri = `${window.location.protocol}//${window.location.host}`;
```